File: nodejs/yandexsearch.js
const response = await fetch(
    'https://scraper-api.smartproxy.com/v2/scrape', {
        method: 'POST',
        body: {
            target: 'yandex',
            parse: false,
            url: 'https://yandex.com/search/?text=cookies&lr=10393'
        },
        headers: {
            'Content-Type': 'application/json',
            'Authorization': 'Basic AUTH'
        },
    }
).catch(error => console.log(error));

console.log(response)

File: python/yandexsearch.py
import requests

task_params = {
    'target': 'yandex',
    'url': 'https://yandex.com/search/?text=cookies&lr=10393',
    'parse': False,
}

username = 'SPusername'
password = 'SPpassword'

response = requests.post(
    'https://scraper-api.smartproxy.com/v2/scrape',
    json = task_params,
    auth = (username, password)
)

print(response.text)
